packages/expo-camera/plugin/src/withCamera.ts

```diff
@@ -14,52 +19,86 @@ const MICROPHONE_USAGE = 'Allow $(PRODUCT_NAME) to access your microphone';
 // Because we need the package to be added AFTER the React and Google maven packages, we create a new allprojects.
 // It's ok to have multiple allprojects.repositories, so we create a new one since it's cheaper than tokenizing
 // the existing block to find the correct place to insert our camera maven.
-const gradleMaven =
-  'allprojects { repositories { maven { url "$rootDir/../node_modules/expo-camera/android/maven" } } }';
+const gradleMaven = [
+  `def mavenPath = new File(["node", "--print", "require.resolve('expo-camera/package.json')"].execute().text.trim(), "../android/maven")`,
+  `allprojects { repositories { maven { url($mavenPath.absolutePath) } } }`,
```

Comment:
I checked it on real project, here's a fixed version that builds successfully 😉 :
```suggestion
  `def mavenPath = new File(["node", "--print", "require.resolve('expo-camera/package.json')"].execute().text.trim(), "../android/maven")`,
  `allprojects { repositories { maven { url(mavenPath) } } }`,
```

Reply:
Awesome, thanks!